views/files/file-viewer.hbs

```diff
@@ -1,16 +1,16 @@
 <section class="file-viewer">
-    <a href="#" class="lightbox videostop" id="file-view">
+    <a href="#" class="lightbox videostop" id="file-view" role="button" aria-label="{{$t "files.label.backToFiles"}}" tabindex="0">
```

Comment:
instead of adding a role and tabindex just use button :-)

Reply:
This would destroy the ability to display videos, as when loading a video, buttons will be added as children nodes to this node which wouldnt be valid html.